docs/sources/reference/commandline/cli.md

```diff
@@ -572,6 +572,12 @@ in cases where the same set of files are used for multiple builds. The path
 must be to a file within the build context. If a relative path is specified
 then it must to be relative to the current directory.
 
+If the docker client loses connection to the daemon, the build is canceled.
+This happens if you interrupt the docker client with `ctrl-c` or if the docker
+client is killed for any reason. (Note: currently only the "run" phase of the
+build can be canceled until pull cancelation is implemented. See
+[Pull 9774](https://github.com/docker/docker/pull/9774) and followups issues.)
```

Comment:
I'm leery of putting future plans into documentation. It's a real maintenance headache. How will we know to remove this note when/if #9774 is completed? We should only describe what is true now: "(Note: only the "run" phase of a build can be canceled.)"